ql/src/test/results/clientpositive/beeline/escape_comments.q.out

```diff
@@ -78,7 +78,6 @@ PREHOOK: Input: escape_comments_db@escape_comments_tbl1
 POSTHOOK: query: describe formatted escape_comments_tbl1
 POSTHOOK: type: DESCTABLE
 POSTHOOK: Input: escape_comments_db@escape_comments_tbl1
-# col_name	data_type	comment
```

Comment:
In this case, header is lost but it does not appear next to values either? Is this correct?

Reply:
In case of the beeline outputs we don't need the header here in the output. So for some reason the output here doesn't contain it, still it will be there in the actual real BeeLine output. When it was present here too, the tables had 2 headers, by removing this row they will only have one. Which is not shown in the test output :) Kind of confusing, we should have a jira for making the TestBeeLineDriver writing characters to the output files exactly as a real BeeLine would.

Reply:
Yeah, I wonder what may be different between the test env of the driver and the real one... Anyway, good to know, thanks.